### PR TITLE
Volunteer Creation Page Updates

### PR DESCRIPTION
**Pull request by supreti17**
Saves data as json to local file, Reads data from saved file, Added
recycler view in home acitvity to display saved json data, Added FAB
button in home activity to start volunteer creation page

### CODE REVIEW

**Comment by scooledge** [comment]
@supreti17  #6 Please use git ignore on the following files: 

CityTrax/.idea/vcs.xml
CityTrax/app/build.gradle

**Comment by scooledge** [comment]
@supreti17 #6 Please use git ignore on the following files:

CityTrax/.idea/vcs.xml
CityTrax/app/build.gradle

Uploading these files causes issues with our personal development environments.

File: CityTrax/.idea/vcs.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (scooledge) [inline CityTrax/.idea/vcs.xml:1]: @supreti17  Remove this file

File: CityTrax/.idea/vcs.xml [hunk 2]
```diff
@@ -1,6 +0,0 @@
...
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
-  </component>
-</project>
```
Comment (scooledge) [inline CityTrax/.idea/vcs.xml:6]: @supreti17 please remove this file. Use git Ignore when committing.

**Review by scooledge — CHANGES_REQUESTED** [review]

File: CityTrax/app/build.gradle [hunk 3]
```diff
@@ -20,7 +20,7 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (scooledge) [inline CityTrax/app/build.gradle]: Again this does not need to be added to the git repo. Use git ignore.

**Review by scooledge — CHANGES_REQUESTED** [review]

**Comment by supreti17** [comment]
Thank you @scooledge. I just removed the files you mentioned from the branch. I'll use git ignore from now on.

**Pull request merged**